marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/auth/WebSecurityConfiguration.java

```diff
@@ -55,12 +59,19 @@ protected void configure(HttpSecurity http) throws Exception {
             .and()
             // Define requests that are always permitted, regardless of whether the user is authenticated or not
             .authorizeRequests()
+                // Needed for websocket tests
+//                .antMatchers("/websocket/**").permitAll()
```

Comment:
Minor nitpick, no need to change, for now, we can get rid of these commented out lines.